queries/query.go

```diff
@@ -117,64 +117,88 @@ func RawG(query string, args ...interface{}) *Query {
 // Exec executes a query that does not need a row returned
 func (q *Query) Exec(exec boil.Executor) (sql.Result, error) {
 	qs, args := BuildQuery(q)
+	before := time.Now()
```

Comment:
Do we need to know the "now" for not debug mode?

Reply:
No,  should I add it under Debug check?

Reply:
I think so. Execs and queries are on hot path so any new allocation could impact performance.